src/os/ObjectStore.h

```diff
@@ -2030,6 +2030,9 @@ class ObjectStore {
   virtual void inject_mdata_error(const ghobject_t &oid) {}
 
   virtual void compact() {}
+  virtual bool has_internal_csum() {
```

Comment:
could you s/internal/builtin/ and mark this method `const`

Reply:
> and mark this method `const`?

@xiexingguo could you address this also?

Reply:
Sure. It slips over my eyes by accident...